04-onion-routing.md

```diff
@@ -201,6 +201,9 @@ This is formatted according to the Type-Length-Value format defined in [BOLT #1]
     1. type: 16 (`payment_metadata`)
     2. data:
         * [`...*byte`:`payment_metadata`]
+    1. type: 18 (`resolution_format`)
+    2. data:
+        * [`byte`:`resolution_format`]
```

Comment:
An alternative for this data field could be to signal the maximum number of hops. This allows senders to trade off privacy (hiding the actual number of hops) and size of the failure package. This avoids picking a constant.